component/src/main/java/io/siddhi/extension/io/grpc/util/GrpcUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c)  2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.siddhi.extension.io.grpc.util;
+
+import io.siddhi.core.exception.SiddhiAppRuntimeException;
+import io.siddhi.query.api.exception.SiddhiAppValidationException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class to hold the static util methods needed
+ */
+public class GrpcUtils {
+    public static String getServiceName(String path) {
+        List<String> urlParts = new ArrayList<>(Arrays.asList(path.split(GrpcConstants.PORT_SERVICE_SEPARATOR)));
+        urlParts.removeAll(Collections.singletonList(GrpcConstants.EMPTY_STRING));
+        if (urlParts.size() < 2) {
+            throw new SiddhiAppValidationException("Malformed URL. After port number atleast two sections should " +
+                    "be available seperated by / as follows grpc://host:port/ServiceName/MethodName");
```

Comment:
Say `by '/' as in 'grpc://<host>:<port>/<ServiceName>/<MethodName>'"` fix other placess too